File: examples/automations/outlook/create.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.outlook.create(
    name="Outlook Automation",
    processor_id="proc_-AnnMLvJs6JwNc5scIWPq",
    webhook_url="https://your-server.com/webhook",
)

print(automation.model_dump_json(indent=2))

File: cookbook/typescript/automations/outlook/create.ts
import { config } from 'dotenv';
import { Retab } from '@retab/node';



async function main() {
  config({ path: '../../../.env.production' });
  
  const reclient = new Retab();
  
  const automation = await reclient.processors.automations.outlook.create({
    name: "Outlook Automation",
    processor_id: "proc_o4dtLxizT0kDAjeKuyVLA",
    webhook_url: "https://your-server.com/webhook",
  });
  
  console.log(JSON.stringify(automation, null, 2));
  
}

main().catch(console.error);